js/app/courseHistoryStore.js

```diff
@@ -82,17 +82,12 @@ const CourseHistoryStore = new GObject.Class({
             tags_match_all: ['EknSetObject'],
             sort: Eknc.QueryObjectSort.SEQUENCE_NUMBER,
         });
-        Eknc.Engine.get_default().query(query, null, (engine, task) => {
-            let results;
-            try {
-                results = engine.query_finish(task);
-            } catch (error) {
-                logError(error);
-                return;
-            }
+        Eknc.Engine.get_default().query_promise(query).then((results) => {
```

Comment:
Yes, we did decide on left-aligned. I can't find the place where we documented that, it was probably in an eos-sdk issue which are erased now.

I dig airbnb's point of staying close to `async`/`await` (they are landing in SpiderMonkey 52). Although that would mean more indentation churn as the above code would look like
```js
try {
    let results = await Eknc.Engine.get_default().query(query);
    blah_blah(results);
} catch(error) {
    logError(error);
}
```